docs/middleware.md

```diff
@@ -382,11 +379,11 @@ Mongoose has both query and document hooks for `deleteOne()`.
 ```javascript
 schema.pre('deleteOne', function() { console.log('Removing!'); });
 
-// Does **not** print "Removing!". Document middleware for `remove` is not executed by default
+// Does **not** print "Removing!". Document middleware for `deleteOne` is not executed by default
```

Comment:
is this statement still true for `deleteOne`? (originally it was for `remove`)

Reply:
This is true for `deleteOne()`, https://github.com/Automattic/mongoose/blob/9572cbd7436f1acb2e95f89f6f1267a1201fe5c1/test/query.middleware.test.js#L287 shows that `pre('deleteOne')` is for queries by default